src/vtquery.cpp

```diff
@@ -219,6 +231,71 @@ std::vector<vtzero::property> get_properties_vector(vtzero::feature& f) {
     return v;
 }
 
+double convert_to_double(value_type value) {
+    // float
+    if (value.which() == 1) {
+        return double(boost::get<float>(value));
+    }
+    // double
+    if (value.which() == 2) {
+        return boost::get<double>(value);
+    }
+    // int64_t
+    if (value.which() == 3) {
+        return double(boost::get<int64_t>(value));
+    }
+    // uint64_t
+    if (value.which() == 4) {
+        return double(boost::get<uint64_t>(value));
+    }
+    return 0.0f;
+}
+
+/// apply filters to a feature
+bool filter_feature(vtzero::feature& feature, std::vector<filter_struct> filters) {
```

Comment:
Return true if a feature should be rejected.  Maybe it should return false as it didn't pass the filter.